debian/kano-init.postrm

```diff
@@ -17,7 +17,7 @@ case "$1" in
         systemctl stop overture
 
         systemctl disable kano-init-boot
-
+        systemctl disable stop-unsupported-rpi-boot
```

Comment:
My only concern is if we could get along with disabling services only during a "remove" operation.
I have implemented it this way, here:

 * https://github.com/KanoComputing/kano-os-loader/blob/master/debian/postrm#L11-L15

I suspect - with no actual proof - that a half-installed system would leave several disabled services.

Reply:
Opened a ticket for this as I suspect we'd need to do a full pass over everything to address this https://trello.com/c/DjIQ8CGY/1780-investigate-reliability-of-disabling-systemd-services-during-postrm-scripts-with-removeupgrade-clause